instr.h

```diff
@@ -0,0 +1,519 @@
+#pragma once
+
+static const cpu_instruction_s instrs[] = {
+    {"NOP", 1, false, false, op_nop},               // 0x00
```

Comment:
You can rewrite with C99-style initialization as following:
```c
static const cpu_instruction_s instrs[] = {
    [0x00] =  {"NOP", 1, false, false, op_nop},  
   ...
```

It would be convenient for shell scripts to generate the table from specific pre-defined file(s).